test/index.js

```diff
@@ -1652,6 +1652,62 @@ describe('list', function() {
 
   });
 
+  describe('takeLast', function() {
+
+    it('is a binary function', function() {
+      eq(typeof S.takeLast, 'function');
+      eq(S.takeLast.length, 2);
+    });
+
+    it('type checks its arguments', function() {
+      assert.throws(function() { S.takeLast([1, 2, 3]); },
+                    errorEq(TypeError,
+                            '‘takeLast’ requires a value of type Number ' +
+                            'as its first argument; received [1, 2, 3]'));
+
+      assert.throws(function() { S.takeLast(0, null); },
+                    errorEq(TypeError,
+                            'The second argument to ‘takeLast’ ' +
+                            'cannot be null or undefined'));
+    });
+
+    it('returns a Nothing if n is greater than collection length', function() {
+      eq(S.takeLast(6, ['a', 'b', 'c', 'd', 'e']), S.Nothing());
+      eq(S.takeLast(6, 'abcde'), S.Nothing());
+    });
+
+    it('returns a Nothing if n is negative', function() {
+      eq(S.takeLast(-0, ['a', 'b', 'c', 'd', 'e']), S.Nothing());
+      eq(S.takeLast(-1, ['a', 'b', 'c', 'd', 'e']), S.Nothing());
+      eq(S.takeLast(-0, 'abcdefg'), S.Nothing());
```

Comment:
Let's use `'abcde'` here as well. The difference between this string literal and the one on the following line is a distraction – it's the number literals that are important.